ihatemoney/templates/statistics.html

```diff
@@ -33,6 +33,12 @@ <h2>{{ _("Expenses by Month") }}</h2>
         {% endfor %}
         </tbody>
         </table>
+        <h2>Graphs</h2>
+        <canvas id="balance" height="400"></canvas>
+        <canvas id="monthlySpending" height="400"></canvas>
```

Comment:
I'm not sure about the fixed width: what happens when we view this from a mobile for instance?